packages/back-end/src/server.ts

```diff
@@ -10,3 +10,17 @@ const server = app.listen(app.get("port"), () => {
 });
 
 export default server;
+
+// App-level error handling
+interface UnhandledRejectionError extends Error {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  promise: Promise<any>;
```

Comment:
Make this a real class that extends Error. And use unknown instead of any.

Reply:
fixed